quality_control_stock/models/stock_picking_type.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# (c) 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    @api.multi
+    def _create_qc_trigger(self):
+        qc_trigger = {
+            # Synchronize all translations
```

Comment:
I think this is a TODO, not a comment, because this doesn't synchronize all translations. In fact, I think the name is not translatable, so we need to do that first.